### ISSUE TITLE
rgbunet: Test shuffling of images and masks for training set, and fix validation set to not do augmentation.

### ISSUE THREAD

**Issue by tdowrick** [UCL/scikit-surgerytf#10]
In GitLab by @MattClarkson on May 2, 2020, 16:46

**Comment by tdowrick** [comment]
In GitLab by @MattClarkson on May 2, 2020, 16:47

changed title from **Remove shuffling of images and masks for training set, and fix validation set to not do augmentation.** to **{+rgbunet: +}Remove shuffling of images and masks for training set, and fix validation set to not do augmentation.**

**Comment by tdowrick** [comment]
In GitLab by @MattClarkson on May 2, 2020, 16:59

mentioned in commit b65b41f5bc61d2df7cb32be7df626187f1251c6f

**Comment by tdowrick** [comment]
In GitLab by @MattClarkson on May 4, 2020, 12:16

changed title from **rgbunet: {-Remove-} shuffling of images and masks for training set, and fix validation set to not do augmentation.** to **rgbunet: {+Test+} shuffling of images and masks for training set, and fix validation set to not do augmentation.**

**Comment by tdowrick** [comment]
In GitLab by @MattClarkson on May 9, 2020, 05:18

mentioned in commit ce071f36183ccd2d869e7a8e51ee56a1b9137120

**Comment by tdowrick** [comment]
In GitLab by @MattClarkson on May 9, 2020, 05:18

mentioned in commit fe521b6c006e9c5a3f428f996f9b2ca0b7fef42b

**Comment by tdowrick** [comment]
In GitLab by @MattClarkson on May 9, 2020, 05:18

mentioned in commit 985dcc4dd73fd913b8197ccabe11f18d302d3184

**Comment by tdowrick** [comment]
In GitLab by @MattClarkson on May 11, 2020, 08:25

closed
